### ISSUE TITLE
Sequence requests that don't include the default conformer get borked bad. 

### ISSUE THREAD

**Issue by gitoliver** [GLYCAM-Web/gems#80]
This bug is on actual. https://github.com/GLYCAM-Web/gems/issues/79 is relevant and might be related/causative/fixable at the same time.

If you build a sequence with multiple rotamers, but don't select whatever rotamer is the "default" in gmml, you get a folder that is named e.g. gt, but the 3D shape is actually gg.

deprecated/sequence/build.py L221. 

This calls gmml and tells it to build the default structure with this logic: 

``` 
if buildState.isDefaultStructure: 
            builder.GenerateSingle3DStructureDefaultFiles(outputDirPath) 
        else: 
            builder.GenerateSpecific3DStructure(gmmlConformerInfo, outputDirPath) 
``` 
But nothing is done to tell gmml to change from it's default 3D structure from e.g. gg to gt if gg wasn't requested. 

[Here is where the isDefaultStructure is set](https://github.com/GLYCAM-Web/gems/blob/485b8eb388f02fa543ededf24cc172203e536cb1/gemsModules/deprecated/sequence/structureInfo.py#L342) 
 
Our test case in 008 doesn't reproduce what the website does so this requires changes to the tests to better reflect what the website does, or the website needs to change, and the tests need to test this "default not selected" scenario.

**Comment by gitoliver** [comment]
I eventually found the cause of the issue is detailed here: https://github.com/GLYCAM-Web/gems/issues/74. So it's not a result of my recent juggle-test branch merge (or no evidence found yet). 

Attempting to fix this has revealed some problems within gems (nevermind the website for now). I've been working with test 008 to figure out what's going on. Here are the additional problems I've found:

- Requesting Evaluation a second time borks everything. I have a tentative fix for this, though knock-on effects are possible.
- Requesting Build3DStructure without first requesting an Evaluation for a sequence triggers a default build, but only a default build. Anything else you requested is ignored. 
- The original bug is ignored for now, gems tests work differently from the website and have their own issues to resolve first. Note that the website never uses Evaluate. It calls build3Dstructure before it hits the options page. The behavior it triggers and the resulting bug seem to be at least partially unrelated to the other bugs I'm finding.
